composer.json

```diff
@@ -27,32 +27,33 @@
 		"docs": "https://docs.typo3.org/p/ttn/tea/main/en-us/"
 	},
 	"require": {
-		"php": "~7.2.0 || ~7.3.0 || ~7.4.0 || ~8.0.0 || ~8.1.0",
-		"typo3/cms-core": "^10.4.11 || ^11.5.2",
-		"typo3/cms-extbase": "^10.4.6 || ^11.5.2",
-		"typo3/cms-fluid": "^10.4 || ^11.5.2",
-		"typo3/cms-frontend": "^10.4 || ^11.5.2"
+		"php": "~7.4.0 || ~8.0.0 || ~8.1.0",
+		"phpspec/prophecy-phpunit": "^2.0",
+		"typo3/cms-core": "^11.5.2 || ^12.0@dev",
+		"typo3/cms-extbase": "^11.5.2 || ^12.0@dev",
+		"typo3/cms-fluid": "^11.5.2 || ^12.0@dev",
+		"typo3/cms-frontend": "^11.5.2 || ^12.0@dev"
 	},
 	"require-dev": {
-		"doctrine/dbal": "^2.13.8",
+		"doctrine/dbal": "^2.13.8 || ^3.3.7",
 		"ergebnis/composer-normalize": "^2.19.0",
 		"friendsofphp/php-cs-fixer": "^3.4.0",
-		"helmich/typo3-typoscript-lint": "^2.5.2",
+		"helmich/typo3-typoscript-lint": "lwolf-php81-support-dev",
 		"jangregor/phpstan-prophecy": "^1.0.0",
 		"php-coveralls/php-coveralls": "^2.5.2",
-		"phpspec/prophecy": "^1.15.0",
+		"phpspec/prophecy": "dev-master",
 		"phpstan/extension-installer": "^1.1.0",
 		"phpstan/phpstan": "^1.8.2",
 		"phpstan/phpstan-phpunit": "^1.1.1",
 		"phpstan/phpstan-strict-rules": "^1.4.2",
-		"phpunit/phpunit": "^8.5.29",
-		"saschaegerer/phpstan-typo3": "^1.1.2",
+		"phpunit/phpunit": "^8.5.29 || ^9.5.23",
+		"saschaegerer/phpstan-typo3": "12.0.x-dev",
 		"seld/jsonlint": "^1.9.0",
 		"squizlabs/php_codesniffer": "^3.7.1",
 		"symfony/yaml": "^4.4.29 || ^5.3.6 || ^6.0",
-		"typo3/cms-fluid-styled-content": "^10.4 || ^11.5.2",
+		"typo3/cms-fluid-styled-content": "^11.5.2 || ^12.0.x-dev",
 		"typo3/coding-standards": "^0.5.5",
-		"typo3/testing-framework": "^6.16.6"
+		"typo3/testing-framework": "^6.16.6 || dev-main"
```

Comment:
Would suggst to only use
```json
"typo3/testing-framework": "~7.0@dev"
```
here. This version is the version for multi core v11/v12 having the new stuff but may have some breaking stuff not in.